src/components/PartyPanelMembersList/PartyPanelMembersList.css

```diff
@@ -1,15 +1,22 @@
 .member-picture {
   width: auto;
+	border-radius: 50%;
+}
+
+.member-picture img {
+	object-fit: cover;
+	border-radius: 50%;
 }
 
 .panel-party-members {
   margin: 0px;
   display: inline-block;
   padding: 10px;
+	background-color: #bcbcbc;
+	border-radius: 10px;
```

Comment:
looks like left padding of the code is inconsistent.  installing prettier to format your code will resolve these going forward.